snapcraft/tests/fixture_setup.py

```diff
@@ -595,47 +598,6 @@ def terminate(self):
         return Popen(args)
 
 
-class FakeSnapd(fixtures.Fixture):
```

Comment:
Would you mind leaving the FakeSnapd classes where they are and do this in a separate branch (that ideally actually merges them)? Here it adds a lot of noise.

Reply:
I'm trying to merge them in a follow up branch, and the easiest step to get there is to move your fake to the lxd test module.

You can take a look at what I'm trying to do here:
https://github.com/elopio/snapcraft/commit/240772d0806b04def52c23a3022172cac85ba08a

Reply:
I don't see the reasoning... but that PR looks lovely, so I'm happy to accomodate.